### PR TITLE
Add ttl command to return time of expiry

### PR DESCRIPTION
**Pull request by mutle**
I recently needed an equivalent of the Redis `ttl` command in `GitHub::KV` and thought I'd open a PR here too in case we want to integrate it into `github-ds` instead of putting it in `lib/github/ds_extensions.rb`.

### CODE REVIEW

File: lib/github/kv.rb [hunk 1]
```diff
@@ -270,6 +270,30 @@ def mdel(keys)
...
+    #
+    #  kv.ttl("foo")
+    #    # => #<Result value: nil>
+    #
+    def ttl(key)
+      validate_key(key)
+
+      Result.new {
+        kvs = GitHub::SQL.results(<<-SQL, :keys => [key], :connection => connection).to_h
+          SELECT `key`, expires_at FROM key_values WHERE `key` IN :keys AND (`expires_at` IS NULL OR `expires_at` > NOW())
```
Comment (jnunemaker) [inline lib/github/kv.rb]: Minor nit: this could probably be a wee bit more simple with a where `key = ...` instead of the `IN ...`. Might also be able to get away with only selecting `expires_at` and using `GitHub::SQL.value` to avoid a hash and hash lookup.

File: test/github/kv_test.rb [hunk 2]
```diff
@@ -166,6 +166,15 @@ def test_setnx_overwrites_expired_key
     assert_equal "bar2", @kv.get("foo").value!
   end
 
+  def test_ttl
```
Comment (jnunemaker) [inline test/github/kv_test.rb]: Another test to test that a key with an expired ttl returns nil would likely also help with regressions. I can add it after merge though.

**Comment by mutle** [comment]
@jnunemaker I'll get your review addressed tomorrow, no need to merge and push out a release before.

**Comment by jnunemaker** [comment]
@mutle I fixed these up quick so you wouldn't have to. Hope that is ok. I'll get it merged.
